src/ELF/Parser.tcc

```diff
@@ -1353,11 +1376,14 @@ void Parser::parse_section_relocations(uint64_t offset, uint64_t size, Section *
         this->binary_->segments().size() == 0) {
       reloc->purpose(RELOCATION_PURPOSES::RELOC_PURPOSE_OBJECT);
     }
+    // FIXME: Else no purpose?
```

Comment:
https://github.com/lief-project/LIEF/blob/master/src/ELF/Relocation.cpp#L39

Reply:
ok, out of curiosity, what is the meaning of this kind of relocation?

Reply:
It's an information to know that this relocation comes from an object file not a *final* object (e.g. library, executable)